ui/app/pages/swaps/fee-card/index.scss

```diff
@@ -228,9 +246,68 @@
   &__bold {
     font-weight: bold;
   }
+
+  &__savings-tooltip {
+    @include H7;
+
+    color: $Grey-500;
+  }
+
+  &__savings-tooltip-card {
+    width: 294.95068359375px;
```

Comment:
this is making the div *bigger* than its allowed screen size... I also don't think it's strictly necessary to make it look good. not a blocker, but these oddly specific pixel values are prone to breaking later.